pkg/bot-github/client.go

```diff
@@ -26,10 +26,16 @@ func NewClient(token string) *Client {
 	}
 }
 
+type CreateBranchArgs struct {
```

Comment:
The changes here for the struct and for the `Client` method are examples of my request, please follow this pattern